common/BasicTypes.asn1

```diff
@@ -25,13 +25,148 @@ BasicTypes DEFINITIONS AUTOMATIC TAGS ::= BEGIN
 
 EXPORTS ALL;
 
--- TODO PTC PFC!
+PUSC-BOOL1 ::= BOOLEAN --PFC-0
+PUSC-BOOL8 ::= BOOLEAN --PFC-8
+PUSC-BOOL16 ::= BOOLEAN --PFC-16
+PUSC-BOOL32 ::= BOOLEAN --PFC-32
 
-PUSC-BOOL ::= BOOLEAN
+PUSC-UINT4 ::= INTEGER (0 .. 15) --PFC-0
+PUSC-UINT5 ::= INTEGER (0 .. 31) --PFC-1
+PUSC-UINT6 ::= INTEGER (0 .. 63) --PFC-2
+PUSC-UINT7 ::= INTEGER (0 .. 127) --PFC-3
+PUSC-UINT8 ::= INTEGER (0 .. 255) --PFC-4
+PUSC-UINT9 ::= INTEGER (0 .. 511) --PFC-5
+PUSC-UINT10 ::= INTEGER (0 .. 1023) --PFC-6
+PUSC-UINT11 ::= INTEGER (0 .. 2047) --PFC-7
+PUSC-UINT12 ::= INTEGER (0 .. 4095) --PFC-8
+PUSC-UINT13 ::= INTEGER (0 .. 8191) --PFC-9
+PUSC-UINT14 ::= INTEGER (0 .. 16383) --PFC-10
+PUSC-UINT15 ::= INTEGER (0 .. 32767) --PFC-11
+PUSC-UINT16 ::= INTEGER (0 .. 65535) --PFC-12
+PUSC-UINT24 ::= INTEGER (0 .. 16777215) --PFC-13
+PUSC-UINT32 ::= INTEGER (0 .. 4294967295) --PFC-14
+PUSC-UINT48 ::= INTEGER (0 .. 281474976710655) --PFC-15
+--! PUSC-UINT64 ::= INTEGER (0 .. 18446744073709551615) PFC-16
+PUSC-UINT64 ::= INTEGER (0 .. 9223372036854775807) -- 2^64-1 value not supported by asn1scc
```

Comment:
should be in newer version

Reply:
Latest is 3.3.22 which I have. Should I compile master or wait for now?

Reply:
I think this commit fixes it: https://github.com/ttsiodras/asn1scc/commit/9d87c9959bef0f12381f34928f7ee041f31fe22b

so it seems that 3.3.23 should be ok